src/GraphQL/Types/BooleanGraphType.cs

```diff
@@ -11,10 +12,11 @@ public BooleanGraphType()
 
         public override object Serialize(object value)
         {
-            if (value is bool)
+            try
             {
-                return (bool) value;
+                return ParseValue(value);
             }
+            catch(Exception) { }
```

Comment:
Let’s not catch any exceptions here and let them bubble up.

Reply:
I had caught the exception because the old Serialize method was always returning the boolean value.

Reply:
Done.